package.json

```diff
@@ -15,7 +15,8 @@
     "docgen": "hardhat docgen",
     "clean": "hardhat clean && hardhat clean --config hardhat.config.zksync.ts",
     "local-test-node:zksyncmainnet": "hardhat node-zksync --fork https://mainnet.era.zksync.io --config hardhat.config.zksync.ts --tag 0.1.0-alpha.31",
-    "local-test-node:zksyncsepolia": "hardhat node-zksync --fork https://sepolia.era.zksync.dev --config hardhat.config.zksync.ts --tag 0.1.0-alpha.31"
+    "local-test-node:zksyncsepolia": "hardhat node-zksync --fork https://sepolia.era.zksync.dev --config hardhat.config.zksync.ts --tag 0.1.0-alpha.31",
+    "local-anvil-node:zksyncsepolia": "anvil-zksync --gas-price 2000000000 fork --fork-url sepolia-testnet"
```

Comment:
I would add `anvil-zksync` as a prerequisite in the README.md file, including a link to https://github.com/matter-labs/anvil-zksync